include "session_event.thrift"

namespace cpp session.server.event


// 公会每日清理事件
// type: EVENT_GUILD_DAILY_CLEAN
struct EventGuildDailyClean {
}


// 公会每周清理事件
// type: EVENT_GUILD_WEEKLY_CLEAN
struct EventGuildWeeklyClean {
}

